src/services/BudgetService.ts

```diff
@@ -107,14 +116,25 @@ export class BudgetService {
     return await QuarterBudgetModel.createNewBudgets(transparencyBudgets)
   }
 
-  static getCurrentBudget(): Promise<CurrentBudget> {
-    return QuarterBudgetModel.getCurrentBudget()
+  static async getCurrentBudget(): Promise<CurrentBudget> {
+    const currentBudget = await QuarterBudgetModel.getCurrentBudget()
+    if (currentBudget !== null) {
+      return currentBudget
+    } else {
```

Comment:
You can remove the else condition as you are doing a return in the if condition.